tests/phpunit/unit-tests/test-pdf.php

```diff
@@ -1903,7 +1903,7 @@ public function test_core_template_options() {
 		$this->assertNotFalse( strpos( $results, 'This is the footer' ) );
 		$this->assertNotFalse( strpos( $results, 'This is the first footer' ) );
 
-		$this->assertNotFalse( strpos( $results, 'background-image: url(/path/image.png) no-repeat 0 0;' ) );
+		$this->assertNotFalse( strpos( $results, 'background-image: url(/path/image.png?' . $data['entry'][16] . ') no-repeat 0 0;' ) );
```

Comment:
Let's hardcode the output of `$data['entry'][16]` rather than make it dynamic. This ensures no unexpected changes to our model.